ui/src/views/mint/form-step/fields/verify-nfa-field.tsx

```diff
@@ -0,0 +1,29 @@
+import { Card, Flex, Grid } from '@/components';
+import { Mint } from '@/views/mint/mint.context';
+
+export const VerifyNFAField = () => {
+  const { verifyNFA, setVerifyNFA } = Mint.useContext();
+
+  const handleVerifyNFAChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setVerifyNFA(e.target.checked);
+  };
+
+  return (
+    <Card.Text css={{ p: '$4', textAlign: 'left' }}>
+      {/* TODO replace for grid */}
+      <Flex css={{ gap: '$10' }}>
+        <Grid css={{ rowGap: '$1h' }}>

```

Comment:
It seems like here could be a single grid, maybe the `Card.Text` could be used as a grid in this case.